src/Board.js

```diff
@@ -1,25 +1,45 @@
 import React, { useState } from "react";
 import Square from "./Square";
-import { TREASURES } from './cards';
+import { TREASURES } from "./cards";
 
 export default function Board() {
-  const randomTreasure = TREASURES[Math.floor(Math.random() * TREASURES.length)].grid;
+  const randomTreasure =
+    TREASURES[Math.floor(Math.random() * TREASURES.length)].grid;
   const [gameBoard, setGameboard] = useState(randomTreasure);
+  const [turn, setTurn] = useState([
+    [1, 1, 1, 1],
+    [1, 1, 1, 1],
+    [1, 1, 1, 1],
+    [1, 1, 1, 1],
```

Comment:
let's reverse this to work with the shape logic
```suggestion
    [0, 0, 0, 0],
    [0, 0, 0, 0],
    [0, 0, 0, 0],
    [0, 0, 0, 0],
```